<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPhoneCtrlWidget</class>
 <widget class="QWidget" name="CPhoneCtrlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPhoneCtrlWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(163, 156, 172);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">.QToolButton
{
	background-color: rgb(33, 155, 179);
	border-top:2px solid rgb(255,255,255);
	border-left:2px solid rgb(255,255,255);
	border-right:2px solid rgb(127,127,127);
	border-bottom:2px solid rgb(127,127,127);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>1000</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../project/DispatherCommunication.qrc">:/img/phone.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_Display_L">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>DDT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>DDT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>DDT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>DDT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>DDT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>DDT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_Display_R">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../project/DispatherCommunication.qrc">:/img/phone.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>38</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">.QToolButton
{
	background-color: rgb(33, 155, 179);
	border-top:2px solid rgb(255,255,255);
	border-left:2px solid rgb(255,255,255);
	border-right:2px solid rgb(127,127,127);
	border-bottom:2px solid rgb(127,127,127);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="Btn_Quit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Help">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>帮助</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Config">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Number">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>拨号盘</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Record">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>呼叫
纪录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_PhoneBook">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>号码簿</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_15">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_18">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_14">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_13">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_17">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pushButton_16">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">.QToolButton
{
	background-color: rgb(159, 171, 147);
	border-top:2px solid rgb(255,255,255);
	border-left:2px solid rgb(255,255,255);
	border-right:2px solid rgb(127,127,127);
	border-bottom:2px solid rgb(127,127,127);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="Btn_LoudSpeaker_L">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>左免提</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Hold">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>保持</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Meeting">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>会议</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Translate">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>转移</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Direct_Call">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>直接
拨号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Pro_Sele">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>程序
选接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Recall">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>重播</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Cancel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Right">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>键权</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Single_Call">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>单呼</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_Mix_Call">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>并机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="Btn_LoudSpeaker_R">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>右免提</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../project/DispatherCommunication.qrc"/>
 </resources>
 <connections/>
</ui>
